docs/language-specification.md

```diff
@@ -68,28 +68,46 @@ with `k` being of type `bit<W>`. Then the fields that can be accessed using
 the `::` operator together with their types are summarized in the following
 table:
 
-| <match_type> | field            | field type |
-|--------------|------------------|------------|
-| exact        | k::value         | bit\<W\>   |
-| ternary      | k::value         | bit\<W\>   |
+| <match_type> | field            | field type | implicit P4Runtime well-formedness constraints | 
+|--------------|------------------|------------|------------------------------------------------|
+| exact        | k::value         | bit\<W\>   | none
+| ternary      | k::value         | bit\<W\>   | masked-off bits are zero: `value & !mask == 0`
```

Comment:
Optional nit: I personally find `value & mask == value` slightly clearer (perhaps because it uses one less operator), but either's fine with me.

Reply:
I like it. But how do we describe that property in English? 
What I like about `value & !mask == 0` is that it closesly matches the English description, "masked-off bits are zero".